recipe/meta.yaml

```diff
@@ -136,33 +114,25 @@ requirements:
     # Otherwise build and host might end up with two different versions
     # If they do, merge_build_host will fail.
     - readline                           # [linux]
+    - llvm                               # [osx]
   host:
     # For pkg-config gl, fontconfig + freetype
     - pthread-stubs                      # [linux]
     - dbus                               # [linux]
     - fontconfig                         # [linux]
     - freetype                           # [linux]
-    - gst-plugins-base                   # [linux]
-    - gstreamer                          # [linux]
+    - glib                               # [not arm64]
+    - gst-plugins-base                   # [not arm64]
+    - gstreamer                          # [not arm64]
+    - pulseaudio                         # [linux and not aarch64]
     - libglib                            # [linux]
     - libxml2                            # [linux]
     - libxkbcommon                       # [linux]
-    # For WebEngine, but for compatiblity we want to restrict
-    # usage of our own libraries. It seems Chromium prefers
-    # expat to libxml2 and we will try to force it to use the
-    # bundled one.
-    # Other than that (and esp. for Google developed libs) we
-    # avoid our own packagess. FFMPEG was not detected anyway,
-    # and neither was libvpx (AFAICT).
-    # - libxml2                            # [unix]
-    # - libxslt                            # [unix]
-    # - libopus                            # [unix]
-    # - libvpx                             # [unix]
-    # - ffmpeg                             # [unix]
     - expat                              # [linux]
     - libevent                           # [linux]
     - icu
-    - jpeg
+    # - jpeg
+    - libjpeg-turbo
```

Comment:
Related issue: https://github.com/conda-forge/conda-forge.github.io/issues/673 .

Reply:
Please don't use libjpeg-turbo until the above issue is resolved.